### PR TITLE
DTSPO-658 Import module application-backend

### PR DESCRIPTION
**Pull request by zc-hmcts**
### JIRA link ###
https://tools.hmcts.net/jira/browse/DTSPO-658

### Change description ###
If applied, this change will import this module https://github.com/hmcts/terraform-module-application-backend

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: azure-pipelines.yaml [hunk 1]
```diff
@@ -10,6 +10,9 @@ parameters:
     displayName: Component to Run
     type: object
     default:
+      - deployment: 'backend'
```
Comment (timja) [inline azure-pipelines.yaml:13]: how about this? to be a bit clearer?
```suggestion
      - deployment: 'backend-app-gateway'
```

File: components/backend/main.tf [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  source      = "git::https://github.com/hmcts/terraform-module-log-analytics-workspace-id.git?ref=master"
+  environment = var.environment
+
+}
+
+
+module "backend" {
+  source = "git::https://github.com/hmcts/terraform-module-application-backend.git?ref=master"
+
+
```
Comment (timja) [inline components/backend/main.tf:11]:
```suggestion
```

File: components/backend/main.tf [hunk 3]
```diff
@@ -0,0 +1,25 @@
+module "logworkspace" {
+  source      = "git::https://github.com/hmcts/terraform-module-log-analytics-workspace-id.git?ref=master"
+  environment = var.environment
+
```
Comment (timja) [inline components/backend/main.tf:4]:
```suggestion
```

File: components/backendappgateway/data_azurerm.tf [hunk 4]
```diff
@@ -0,0 +1 @@
+data "azurerm_subscription" "current" {}
```
Comment (msl8r) [inline components/backendappgateway/data_azurerm.tf:1]: do we need this?

File: components/backendappgateway/main.tf [hunk 5]
```diff
@@ -0,0 +1,25 @@
+module "logworkspace" {
+  source      = "git::https://github.com/hmcts/terraform-module-log-analytics-workspace-id.git?ref=master"
+  environment = var.environment
+}
+
+
+module "backendappgateway" {
+  source = "git::https://github.com/hmcts/terraform-module-application-backend.git?ref=DTSPO-658-fix-features-issue"
```
Comment (msl8r) [inline components/backendappgateway/main.tf:8]:
```suggestion
  source = "git::https://github.com/hmcts/terraform-module-application-backend.git?ref=master"
```

Comment (msl8r) [reply components/backendappgateway/main.tf:8]: now we've merged to master let's set tf to use that branch

**Review by msl8r — APPROVED** [review]
LGTM 🌟

**Pull request merged**